R/transformCounts.R

```diff
@@ -551,7 +551,7 @@ setMethod("relAbundanceCounts", signature = c(x = "SummarizedExperiment"),
 .apply_pseudocount <- function(mat, pseudocount){
     if( .is_a_bool(pseudocount) ){
         # If pseudocount TRUE and some negative values, numerical pseudocount needed
```

Comment:
You could test that if there are NA values, I don't know what to do with them so better to disable pseudocount=TRUE if there are NAs

Reply:
or hmmm, maybe it is not necessary since after adding pseudocount the NAs are still NAs

Reply:
But calculate pseudocount with ifelse(pseudocount, min(mat[mat>0], na.rm = TRUE), 0)

Reply:
Or hmm, I think the best choice is to disable this when NAs...